scripts/index.js

```diff
@@ -1 +1,2 @@
-import './header.js'
+import './header.js';
```

Comment:
@Tara-Tiong @Ernesto2004 @kingtrentiii let's continue not using semicolons for JS:

https://blog.izs.me/2010/12/an-open-letter-to-javascript-leaders-regarding
http://inimino.org/~inimino/blog/javascript_semicolons

TLDR -- Javascript does something funny called "automatic semicolon insertion," and it does it even if you put semicolons in, so the ones you add yourself are useless 99% of the time -- and potentially confusing because they don't prevent ASI from taking place.

Like most style things, it's a topic of debate, but nowadays my view is that it's best to just use them only where they'd actually change the meaning, so that the code does exactly what it says (once you take JS's ASI behavior into account).